test/step-function.test.ts

```diff
@@ -4019,6 +4132,37 @@ describe("binding", () => {
     expect(normalizeDefinition(definition)).toMatchSnapshot();
   });
 
+  // TODO: https://github.com/functionless/functionless/issues/420
+  // https://mathiasbynens.be/notes/javascript-properties
+  test.skip("unicode variable names", () => {
```

Comment:
This test is so awesome. Shame we skip it

Reply:
Just need to go the ticket to make sure it is all supported. Mostly need to normalize to valid characters in ASL.